api/v1alpha1/commons.go

```diff
@@ -10,7 +10,7 @@ import (
 type KubeAuthConfiguration struct {
 	// ServiceAccount is the service account used for the kube auth authentication
 	// +kubebuilder:validation:Required
-	// +kubebuilder:default={Name:&#34;default&#34;}
+	// +kubebuilder:default={Name:default}
```

Comment:
this was intentional and needed because that how you do default for a structure.

Reply:
The generator was complaining about unexpected characters.

Reply:
from the docs:
```
A default value will be accepted as any value valid for the field. Formatting for common types include: boolean: true, string: Cluster, numerical: 1.24, array: {1,2}, object: {policy: &#34;delete&#34;}). Defaults should be defined in pruned form, and only best-effort validation will be performed. Full validation of a default requires submission of the containing CRD to an apiserver.
```